config.py

```diff
@@ -0,0 +1,47 @@
+import os
+from dataclasses import dataclass
+
+from dotenv import load_dotenv
+
+# Default values as constants
+DEFAULT_LOGGING_LEVEL = "DEBUG"
+DEFAULT_LOGGING_FORMAT = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+DEFAULT_SESSION_LIFETIME = 300
+DEFAULT_TOKEN_EXPIRES_IN_SECONDS = 300
+DEFAULT_APP_ID = "flask-example"
+DEFAULT_REDIRECT_PATH = "/"
+
+load_dotenv()
+
+
+def read_key(membrane_server, env):
+    if membrane_server:
+        return open(os.getenv(env), "rb").read()
```

Comment:
https://docs.quantifiedcode.com/python-anti-patterns/maintainability/not_using_with_to_open_files.html

can you investigate if one of the linter could have caught this?

https://code.visualstudio.com/docs/python/linting

Reply:
Not out of the box. They would need advanced configuration.